experiment/telegram/telegram_otherwise.go

```diff
@@ -0,0 +1,184 @@
+// +build !cgo
+
+// Package telegram contains the Telegram network experiment. This file
+// in particular is a pure-Go implementation of that.
+//
+// See https://github.com/ooni/spec/blob/master/nettests/ts-020-telegram.md.
+package telegram
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/ooni/netx/handlers"
+	"github.com/ooni/netx/httpx"
+	"github.com/ooni/probe-engine/experiment"
+	"github.com/ooni/probe-engine/experiment/handler"
+	"github.com/ooni/probe-engine/log"
+	"github.com/ooni/probe-engine/model"
+	"github.com/ooni/probe-engine/session"
+)
+
+const (
+	testName    = "telegram"
+	testVersion = "0.0.2"
```

Comment:
I suggest bumping this up to a number different than version being used in measurement-kit so we can more easily distinguish them.

Reply:
Yes, good suggestion.

Reply:
done!